tee-worker/ts-tests/stress-runner

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/bash
+
+mkdir -p stress-output
```

Comment:
TODO: take the number of processes and output dir as a parameter; set correct working directory.

Reply:
dropped for now